src/DynamoCoreWpf/UI/GuidedTour/HostInfo.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Windows;
+using System.Windows.Controls.Primitives;
+
+namespace Dynamo.Wpf.UI.GuidedTour
+{
+    /// <summary>
+    /// This class will contain info related to the popup location and the host control in which the popup will appear over
+    /// </summary>
+    public class HostInfo
```

Comment:
Maybe we can rename this to `HostControlInfo` or something you think is better. Host alone in Dynamo means Dynamo integrator so this class is a bit misleading IMHO

Reply:
@QilongTang I renamed the class to HostControlInfo, so I had to do the renaming in several places.